src/kmipapi/clientapi.go

```diff
@@ -106,7 +106,8 @@ func DiscoverServer(ctx context.Context, settings *ConfigurationSettings, client
 	}
 
 	req := DiscoverRequest{
-		ClientVersions: clientVersions,
+		// Need to comment out for Hashicorp to work in 1.4
+		//ClientVersions: clientVersions,
```

Comment:
```suggestion
		ClientVersions: clientVersions,
```

Reply:
We don't want to put into the common low-level layers specific handlers for KMIP Servers. Instead we will need to put this into the user of this library.